freqtrade/configuration/cli_options.py

```diff
@@ -255,6 +255,12 @@ def __init__(self, *args, **kwargs):
         help='Print all exchanges known to the ccxt library.',
         action='store_true',
     ),
+    # List timeframes
+    "list_timeframes_print_one_column": Arg(
+        '-1', '--one-column',
+        help='Print timeframes in one column.',
```

Comment:
wouldn't "Show/print output in one column." also solve this without duplicating this?

You're running "list-timeframes" already, so do we really need to mention this again ... ?

Reply:
I made it this way initially, but did not like the common string (not this one) I printed there and decided that two different descriptions would be better.